website/docs/api/cli.mdx

```diff
@@ -1699,6 +1702,44 @@ $ python -m spacy project dvc [project_dir] [workflow] [--force] [--verbose] [--
 | `--help`, `-h`    | Show help message and available arguments. ~~bool (flag)~~                                                    |
 | **CREATES**       | A `dvc.yaml` file in the project directory, based on the steps defined in the given workflow.                 |
 
+## distill {id="distill", tag="experimental", version="4"}
+
+Distill a _student_ pipeline from a _teacher_ pipeline. Distillation trains the
+models in the student pipeline on the activations of the teacher's models. A
+typical use case for distillation is to extract a smaller, more performant model
+from large high-accuracy model. Since distillation uses the activations of the
+teacher, distillation can be performed on a corpus without (gold standard)
+annotations.
+
+`distill` will save out the best model from all epochs, as well as the final
```

Comment:
```suggestion
`distill` will save out the best performing pipeline across all epochs, as well as the final
```
Mostly to distinguish "model" from "pipeline" - the eval is holistic across all models in the one pipeline